operations/src/main/java/com/nextdoor/bender/operation/filter/FilterOperationFactory.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * Copyright 2018 Nextdoor.com, Inc
+ *
+ */
+
+package com.nextdoor.bender.operation.filter;
+
+import com.nextdoor.bender.config.AbstractConfig;
+import com.nextdoor.bender.operation.Operation;
+import com.nextdoor.bender.operation.OperationFactory;
+
+/**
+ * Create a {@link FilterOperation}.
+ */
+public class FilterOperationFactory implements OperationFactory {
+
+  private FilterOperationConfig config;
+
+  @Override
+  public Operation newInstance() {
+    return new FilterOperation(
```

Comment:
Compile the regex here and pass in `Pattern` to your operation. Eg https://github.com/Nextdoor/bender/blob/master/operations/src/main/java/com/nextdoor/bender/operation/json/key/KeyNameReplacementOperationFactory.java#L48